doc/en/user/source/extensions/css/workshop/index.rst

```diff
@@ -0,0 +1,16 @@
+.. _extensions_css_cartography_styling:
+
+********************
+CSS Styling Workshop
+********************
+
+GeoServer styling can be used for a range of creative effects. This section introduces the `CSS Extension` which can be used to quickly generate SLD files.
```

Comment:
I've changed it to follow immediately after the tutorial as you suggested and called it an expansion/more in-depth look at styling. How does that sound?